index.html

```diff
@@ -7,14 +7,71 @@
     <title>Zürcher IMS Alumni Netzwerk</title>
     <meta name="description" content="IMS Alumni Netzwerk">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+        body {
+            margin: 0;
+            font-family: Roboto, RobotoDraft, Helvetica, Arial, sans-serif;
+        }
+
+        iframe {
+            width: 100%;
+            height: 1500px;
+            border: none;
+            overflow: hidden;
```

Comment:
we need to check that it also works on mobile phones :) maybe it is safer to write

```css
min-height: 1500px;
overflow: scroll;
```